recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
```

Comment:
Since changes to the maintainer list doesn't change the package contents, you can leave the build number at `0`.
Other than that, LGTM, and thanks for volunteering as a maintainer!

Reply:
Thanks! I wasn't sure about it.